packages/osfamilymap.yaml

```diff
@@ -27,3 +27,12 @@ RedHat:
       pkgs:
         - rubygems
 
+FreeBSD:
+  pkgs:
```

Comment:
I think this like above should be `pip:` instead of `pkgs`

Reply:
Ok.